apkid/rules/elf/protectors.yara

```diff
@@ -162,3 +162,45 @@ rule vkey_elf : protector
   condition:
     is_elf and $libname and 1 of ($vos*) and 1 of ($detection*) and 1 of ($jni*)
 }
+
+rule verimatrix_elf_arm64 : protector
+{
+  meta:
+    description = "InsideSecure Verymatrix"
+    url         = "https://www.verimatrix.com/products/app-shield/"
+    sample      = "105a2646e8acf45afcd606f4d47af68a8218e0ae7d8c9646d0048cbf32df7a73"
+    author      = "FrenchYeti"
+    
+  strings:
+    // byte sequence from .data, used into JNI_OnLoad
+    $seq = {
+      ?? ?? ?? ?? 88 98 a8 b8
+      ?? ?? ?? ?? 94 a4 b4 c4
+      ?? ?? ?? ?? 88 98 a8 b8
+      ?? ?? ?? ?? 94 a4 b4 c4
+    }
+    
+    $comm1 =  "Android (7019983 based on r365631c3) clang version 9.0.9 (https://android.googlesource.com/toolchain/llvm-project a2a1e703c0edb03ba29944e529ccbf457742737b) (based on LLVM 9.0.9svn)"
```

Comment:
Can we use regexes here?

Reply:
I think it is not a good idea because this pattern appears sometime into lib not protected by Verimatrix, as 
`"Android (6454773 based on r365631c2) clang version 9.0.8 (https://android.googlesource.com/toolchain/llvm-project 98c855489587874b2a325e7a516b99d838599c6f) (based on LLVM 9.0.8svn)"`

Reply:
If you create another rule less strict to hunt more samples, please use regex over there

Reply:
Pattern removed. Rule will be updated later if required, after more tests if some false positive occur